packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -125,51 +119,58 @@ DialogClose.displayName = DialogPrimitive.Close.displayName;
 const DialogHeader = ({
   className,
   size,
+  children,
+  padding = true,
   ...props
-}: React.HTMLAttributes<HTMLDivElement> & SizeProp) => (
-  <>
-    <DialogClose position="right" size={size}>
-      <IconButton name="cross" variant="ghost" />
+}: DialogProps & SizeProp & PropsWithChildren) => (
+  <div
+    className={twMerge(
+      "flex items-center justify-between",
+      padding && defaultPaddingClassName,
+      className,
+      size
```

Comment:
Something like this is what I mean in the comment above this one
```suggestion
      "flex items-center justify-between p-3 md:p-4",
      className,
      size
```